docs/howto.rst

```diff
@@ -44,6 +44,13 @@ You can for instance apply a cone search as shown `here
 You can also select observations based on other informations available using the
 `~gammapy.data.ObservationTable.select_observations` method.
 
+Make a on-axis equivalent livetime map
++++++++++++++++++++
```

Comment:
Please check that the number of '+'  is equal to the number of character in the title.